packages/css/src/identifier.ts

```diff
@@ -75,7 +75,7 @@ export function generateIdentifier(
       packageName,
     });
 
-    if (!identifier.match(/^[A-Z_][0-9A-Z_]+$/i)) {
+    if (!identifier.match(/^[A-Z_][0-9A-Z_\-]+$/i)) {
```

Comment:
The hyphen doesn't need to be escaped when it's first/last in a range.

```suggestion
    if (!identifier.match(/^[A-Z_][0-9A-Z_-]+$/i)) {
```